src/types.ts

```diff
@@ -0,0 +1,19 @@
+export type UserSitter = {
```

Comment:
```suggestion
import { ObjectId } from 'bson';

export type UserSitter = {
_id?: ObjectId;

```
Could be useful to generate an ID from MongoDB